cmd/main.go

```diff
@@ -63,10 +63,10 @@ func main() {
 		}
 		os.Exit(0)
 	}
-	klog.InfoS("Starting Azure Key Vault Provider", "version", version.BuildVersion)
+	klog.V(5).InfoS("Starting Azure Key Vault Provider", "version", version.BuildVersion)
 
 	if *enableProfile {
-		klog.InfoS("Starting profiling", "port", *profilePort)
+		klog.V(5).InfoS("Starting profiling", "port", *profilePort)
```

Comment:
ditto (keep it at level 0)

Most of the logs in main are only logged at initialization and shouldn't cause log flooding